### ISSUE TITLE
Accessibility issue: <li> elements must be contained in a <ul> or <ol> on /vet/Leptospirosis2021.htm

### ISSUE THREAD

**Issue by Alexz1008** [Alexz1008/LACODPHTest#5]
## What
  An accessibility scan flagged the element `<li><a href="https://www.211la.org/" target="_blank"><b>Call 2-1-1 for guidance to LA County health &amp; human services programs.</b></a></li>` on http://www.ph.lacounty.gov/vet/Leptospirosis2021.htm because <li> elements must be contained in a <ul> or <ol>. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/listitem?application=playwright.

  
  To fix this, ensure <li> elements are used semantically.
  
Specifically:

Fix any of the following:
-   List item does not have a <ul>, <ol> parent element

  ## Acceptance Criteria
  - [ ] The specific axe violation reported in this issue is no longer reproducible.
  - [ ] The fix MUST meet WCAG 2.1 guidelines OR the accessibility standards specified by the repository or organization.
  - [ ] A test SHOULD be added to ensure this specific axe violation does not regress.
  - [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
